runtime/onert/core/src/compiler/pass/PermutationEliminationPass.cc

```diff
@@ -16,6 +16,7 @@
 
 #include "PermutationEliminationPass.h"
 
+#include "backend/Backend.h"
```

Comment:
Is it relevant to this PR?

Reply:
Yes. This file needs it by changing the include file from `LoweredGraph.h` to `ILoweredGraph.h` in `LoweredOperandPass.h`